doc/dh_virtualenv.1.rst

```diff
@@ -45,6 +45,7 @@ OPTIONS
 --preinstall=PACKAGE			Preinstall a PACKAGE before
 					running pip.
 --extra-pip-arg				Extra arg for the pip executable.
+--extra-virtualenv-arg      Extra arg for the virtualenv executable.
```

Comment:
Whops, the indentation looks bit off here. Mind fixing the previous line too, even though that's my bad :p